app/src/main/java/com/worldtreeinc/leaves/activity/RoleOptionActivity.java

```diff
@@ -54,9 +57,14 @@ public void onClick(View view) {
             case R.id.bidderBtn:
                 destinationIntent = new Intent(this, BidderDashActivity.class);
                 break;
+            case R.id.logout:
+                User.logout();
+                destinationIntent = new Intent(this, WelcomeActivity.class);
```

Comment:
NITPICK: destinationIntent = new Intent(this, WelcomeActivity.class) was repeated in line 55 and 58. You could DRY this up with a single method.